.storybook/storybook-data.js

```diff
@@ -95,6 +95,22 @@ module.exports = [
 			},
 		},
 	},
+	{
```

Comment:
Thanks, @techanvil. This PR looks good to me, but could you please add a VR test for the zero-data story of the Overview widget in the AdSense module too to make sure that all overview widgets have their zero-data states covered? https://github.com/google/site-kit-wp/blob/0f7293812b3c9a1ce2f023c2cb70c31becf38fc1/.storybook/storybook-data.js#L68-L82

Reply:
Hi @eugene-manuilov, thanks for the review. I discovered during implementation that the AdSense Overview widget does not have a zero-data story in the same way the Search Console and Analytics Overview widgets do. I did mention this in the **Relevant technical choices** section at the top of this PR.

![image](https://user-images.githubusercontent.com/18395600/144076630-3879c7c2-7d2c-472a-9b73-54888519b683.png)

I actually tried adding a **Zero Data** story, but discovered that the AdSense Overview widget renders the view shown in the story **Data Unavailable** when the data is zeroed, due to these lines:

https://github.com/google/site-kit-wp/blob/8bb0a20a1306d8ad4f598f377b528c970bbcbd47/assets/js/modules/adsense/components/module/ModuleOverviewWidget/index.js#L143-L155

So, I figured I'd simply leave the AdSense Overview widget out of this PR. I would however be happy to revisit this choice. What do you think, given the above?

Reply:
Ah... sorry, I didn't notice your comment in the relevant technical choices section... :sweat_smile: That makes sense then. Let's not add it for the AdSense module. :+1:

Reply:
No problem, and thanks for the merge!